hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/TestRDBStore.java

```diff
@@ -151,6 +151,23 @@ public void compactDB() throws Exception {
 
   }
 
+  @Test
+  public void compactTable() throws Exception {
+    assertNotNull(rdbStore, "DBStore cannot be null");
+
+    for (int j = 0; j <= 20; j++) {
```

Comment:
I have an interesting observation here.

When I play with the number of insertions. I noticed that `metaSizeBeforeCompact` always fluctuates between 2 to 5.

It looks like once it is about to reach 6, it does some compaction **before** the new insertion.